### ISSUE TITLE
Maybe add in list Bluecord and Aliucord?

### ISSUE THREAD

**Issue by Glazami** [revancedapps/revanced.net#11]
https://github.com/bluemods/Bluecord
https://github.com/Aliucord/Aliucord

**Comment by revancedapps** [comment]
Yes, we can, but I'll need some time to explore this app, did not use it before

**Comment by Glazami** [comment]
Great 🤝👍
And yet, in the manager I see only 16 applications

**Comment by TheToto** [comment]
Aliucord and Bluecord is based on an old Discord version.
Vendetta is another Discord mod for the new version.

**Comment by Glazami** [comment]
Do you want applications to receive updates from their original author repositories? Updating this repository by manually adding updates for each application causes a lot of delay and wastes a lot of time. it would be easier to get updates from native repositories.

**Comment by revancedapps** [comment]
Thank you. In fact, I have a CI/CD tool that runs every morning to automatically get versions of the patcher. However, before releasing it to [revanced.net](https://revanced.net/), I usually run it on my own phone to make sure that the built version does not have any issues when widely published.
In the past three days, the ReVanced team has just changed the default package of MicroG, and created a new fork from GmsCore, so I'm waiting for it to stabilize a bit more before publishing.
